packages/panels/src/components/sidebar-menu/SidebarMenu.module.css

```diff
@@ -1,15 +1,34 @@
 .sidebarMenu {
   /* Theme vars */
+  --swui-sidebar-menu-item-height: 48px;
   --swui-sidebar-menu-separator-color: var(--lhds-color-blue-700);
   --swui-sidebar-menu-text-color: var(--swui-white);
   --swui-sidebar-menu-icon-color: var(--swui-white);
-  --swui-sidebar-menu-background-color: var(--lhds-color-blue-900);
-  --swui-sidebar-menu-background-color-hover: var(--lhds-color-blue-800);
-  --swui-sidebar-menu-background-color-active: var(--lhds-color-blue-800);
-  --swui-sidebar-menu-item-selected: var(--lhds-color-blue-500);
+
+  --swui-sidebar-menu-background-color: var(--lhds-color-blue-500);
+  --swui-sidebar-menu-background-color-hover: var(--lhds-color-blue-700);
+  --swui-sidebar-menu-background-color-active: var(--lhds-color-blue-700);
+  --swui-sidebar-menu-item-selected: var(--lhds-color-blue-600);
   --swui-sidebar-menu-content-padding: var(--swui-metrics-indent)
     var(--swui-metrics-indent) var(--swui-metrics-indent) 0;
 
+  --swui-sidebar-background-dark-color: var(--lhds-color-blue-800);
```

Comment:
Maybe let the `dark` version be a CSS-module that can be applied? So that it is basically a theme, just like we can change color in buttons and other elements.

SidebarMenuThemes.css

```
  // Override some of the component colors
  .dark {
    --swui-sidebar-menu-background-color: var(--lhds-color-blue-500);
    --swui-sidebar-menu-background-color-hover: var(--lhds-color-blue-700);
    --swui-sidebar-menu-background-color-active: var(--lhds-color-blue-700);
    --swui-sidebar-menu-item-selected: var(--lhds-color-blue-600);
  }
```

```
<SidebarMenu className={sidebarMenuTheme.dark}>
```